app/views/layouts/preview/media.html.erb

```diff
@@ -2,7 +2,6 @@
 <html>
   <head>
     <%= stylesheet_link_tag 'media.css' %>
-    <%= stylesheet_link_tag 'sul_icons.css' %>
```

Comment:
Not sure you can remove this from media? https://github.com/sul-dlss/sul-embed/blob/a5b2b7f1719a00bfe3fa3a73de5b3347706eb8cb/app/components/media/wrapper_component.rb#L25. Might be wrong because this is a layout and there might be something I don't know about.

Reply:
Yeah, this was part of the legacy viewer.  We can remove this too: https://github.com/sul-dlss/sul-embed/commit/41ee4ba342d43ee9b1207b0124f605ebfce6ecfa#diff-85f540687348bc19839e05c7d58a6b7df20886ac2bd6f4617e02c0e2b2ee6ccbR51

Reply:
Can we remove this line in #2357 so they are paired?

Reply:
It ends up messing up the preview layout. It gets rid of the icons in the sidebar.
![Screenshot 2025-02-06 at 11 28 23 AM](https://github.com/user-attachments/assets/fd9519ed-02b3-4a7c-93e6-345a3cc4dc23)

Reply:
Ah! Good catch there.